web/package.json

```diff
@@ -28,7 +28,9 @@
     "build": "react-scripts build",
     "test": "react-scripts test",
     "eject": "react-scripts eject",
-    "lint": "gts check",
+    "lint": "gts check && yarn lint:css",
```

Comment:
I think this one should standardise, either `yarn lint:code && yarn lint:css` or `gts check && stylelint ./src/**/*.tsx`. I think the first one is better cos you don't have to change twice if you change anything.

Reply:
Oh yea! Didn't realise this haha thanks!